api/base_controller.go

```diff
@@ -193,12 +193,18 @@ func (c *BaseController) ListObjects(r *web.Request) (*web.Response, error) {
 		return nil, util.HandleStorageError(err, string(c.objectType))
 	}
 
+	page := types.ObjectPage{
+		Items: make([]types.Object, 0, objectList.Len()),
+	}
+
 	for i := 0; i < objectList.Len(); i++ {
+		page.ItemsCount++
```

Comment:
another approach could be to just change the json tag on the objectlist generator to `json:"items"` and add an Items() method to the ObjectList interface and the generator